Signal/src/view controllers/SignalAttachment.swift

```diff
@@ -147,6 +156,12 @@ class SignalAttachment: NSObject {
     // Returns the file extension for this attachment or nil if no file extension
     // can be identified.
     var fileExtension: String? {
+        if dataUTI == SignalAttachment.kOversizeTextAttachmentUTI ||
+        dataUTI == SignalAttachment.kUnknownTestAttachmentUTI {
+            assert(false)
+            return ""
```

Comment:
nit: What's the distinction between returning "" and returning nil? It seems like a distinction which could surprise the caller.